forecast_tools.R

```diff
@@ -50,6 +50,90 @@ plot_data = function(data, title, observed) {
   plot(sp_predict)
 }
 
+
+#' Compares forecasts to observations over different lead times.
+#' Error can be any function. The level, species, and currency columns from
+#' observations and forecasts must have matching values.
+#' 
+#' Will only return values where there are matching comparison columns (currency, level, species)
+#' 
+#' @param observations dataframe Has the columns NewMoonNumber, currency, level, species, actual
+#' @param forecasts dataframe passes the forecast validity check. Must have matching values in
+#'                  the comparison columns
+#' @param error_metric chr either 'mse' for mean squared error or 'likelihood' for the likelihood of the observation
+#' assumring the estimate and PI's fit a normal distribution
+#' @param ci_value int The value of the forecast confidence interval to scale PI values for the likelihood metric
```

Comment:
I don't see this parameter getting used and am unclear on exactly what it is based on the description.